build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -5,6 +5,80 @@ metadata:
   name: flow-aggregator
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flow-aggregator-ca
+  namespace: flow-aggregator
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: flow-aggregator-client-tls
+  namespace: flow-aggregator
+type: kubernetes.io/tls
+data:
+  tls.crt: ""
+  tls.key: ""
```

Comment:
out-of-curiosity, is this required when setting the Secret type to `kubernetes.io/tls`?

Reply:
Correct, these two fields are required although the API server doesn't actually validate the values for each key. reference: https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets

Reply:
However, I guess re-applying the deployment yaml will override the secret if you specify `data`, `tls.crt`, `tls.key`field here. This is because declaring them here means they are managed by "kubectl-client-side-apply", so after the aggregator sets them, they will still be overriden by `kubectl apply` when updating or upgrading yaml. That's why antrea-ca configmap doesn't have `data` and `ca.crt` set.

I guess you may either change the type to Opaque or remove the secret from yaml and grant the controller permission of creating secret to solve it.

Reply:
It makes sense to me. Thanks. I removed the secret form the yaml considering that users may think this secret is editable.